ui/raidboss/data/04-sb/alliance/royal_city_of_rabanastre.txt

```diff
@@ -13,26 +13,26 @@ hideall "--start--"
 
 ### Mateus
 0.0 "--start--" sync /:The Crumbling Bridge will be sealed off/ window 0,1
-23.0 "Unbind" sync /:Mateus, The Corrupt starts using Unbind/ window 23,10 # start of cast
+23.0 "Unbind" sync /:2633:Mateus, The Corrupt:/ window 23,10 # start of cast
```

Comment:
This is close, but you need to drop the `:` at the end.  (I wonder if `npm run test` will complain at you about this.)

Here's an example of a sync for a `14` and a `15` line for the same ability:
https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/05-shb/raid/e4s.txt#L17-L18

For succinctness, timelines either are:
* 14 lines, `:ability id:name` (no trailing colon, because " starts using ability name" comes directly after)
* 15/16 lines, `:name:ability id:` (trailing colon)

Reply:
`npm run test` will not throw an error